profiles/markdown/config.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "../../schema/jinks.json",
+    "type": "feature",
+    "id": "https://e-editiones.org/app/tei-publisher/markdown",
+    "version": "1.0.0",
+    "label": "Enable Markdown Output via ODD",
+    "description": "This enables the markdown output mode for ODDs, allowing you to render TEI documents as markdown.",
+    "order": 0,
+    "defaults": {
+        "media": ["markdown"]
```

Comment:
This one needs to be moved to base10, to ensure the markdown ODD is generated, which looks like is seen as required by TP5.